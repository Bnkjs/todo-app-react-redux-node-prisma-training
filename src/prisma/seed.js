const { PrismaClient } = require('@prisma/client')
var faker= require('faker');

const prisma = new PrismaClient()

const main = async () => {
  let x = 0
  while(x < 5){
    await prisma.post.create({
      data:{
        id: x,
        title: faker.commerce.productName(),
        completed: false
      }
    })
    x++;
  }
}

main().catch(e => {
  console.log(e);
  process.exit(1)
}).finally(()=>{
  prisma.$disconnect()
})