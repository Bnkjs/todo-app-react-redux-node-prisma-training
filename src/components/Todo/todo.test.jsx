import { render, fireEvent } from "@testing-library/react";
import { Todo } from ".";
import React from 'react';
import { PrismaClient } from ".prisma/client";
import renderer from 'react-test-renderer';
// test(' mockClick call length to be 1', () => {
//   const mockClick = jest.fn()
//   render(<Todo/>)
//   const click = document.querySelector('.todo-el')
//   fireEvent.click(click)
//   expect(mockClick.call.length).toBeCloseTo(1)
// })


it('rend correctement', () => {
  const component = renderer
  .create(<Todo/>)
    let tree = component.toJSON()
  expect(tree).toMatchSnapshot();
});