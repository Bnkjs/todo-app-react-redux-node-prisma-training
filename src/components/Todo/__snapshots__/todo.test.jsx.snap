// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rend correctement 1`] = `
<div
  id="container-input"
>
  <input
    className="todo-el"
    type="checkbox"
  />
  <span>
    To Do: 
  </span>
</div>
`;
